### ISSUE TITLE
Mongoose.connect miss catch error statment

### ISSUE THREAD

**Issue by MartinBucko** [feathers-plus/cli#14]
### Steps to reproduce

I ship my app first time to production server and accidentally setup bad mongodb config URL. In logs it don't show nothing only message `Unhandled Rejection at: Promise ..`. It was hard to find the problem in bad password setup in mongodb URL. After this i found `mongoose.connect` don't have catch statement.

### Expected behavior

`mongose.connect` should throw error after connection fail

In generated file from cli `src/mongoose.js` should be:

```
mongoose
    .connect(app.get('mongodb'), {})
    .catch(err => {
      console.error('Mongoose connection error: ', err)
    })
```

**Module versions** (especially the part that's not working):

feathers-plus cli version 0.4.23

**Comment by eddyystop** [comment]
Thank you.

**Comment by eddyystop** [comment]
A basic goal of the generator is to remain source compatible with feathersjs.cli, which does not log such connection failures. 

I've asked around and found that some people do modify the feathersjs/cli code. I'm proposing the code be changed to:
```js
  mongoose.Promise = global.Promise
  mongoose.connect(app.get('mongodb'), {})
  .then(({ connection }) => {
    console.log(`connected to "${connection.name}" database at ${connection.host}:${connection.port}`)
    return connection
  })
  .catch(error => {
    console.log(error)
    process.exit(0)
  })

  app.set('mongooseClient', mongoose)
```

**Comment by eddyystop** [comment]
The TS version needs
```js
.then(({ connection }: any) => {
```

**Comment by eddyystop** [comment]
Published in @feathers-plus/cli v0.4.24 using generator v0.2.30.
